lib/actions-oclif/os/configure.ts

```diff
@@ -77,6 +83,11 @@ export default class OsConfigureCmd extends Command {
 		The --device-type option may be used to override the application's default
 		device type, in case of an application with mixed device types.
 
+		The --system-connection option can be used to inject a
+		NetworkManager system-connection config file, for providing cellular/network
+		settings. It may be passed multiple times in case there are multiple files
+		to inject.
```

Comment:
Suggestion:
```
		The --system-connection (-c) option can be used to inject NetworkManager connection
		profiles for additional network interfaces, such as cellular/GSM or additional
		WiFi or cable connections. This option may be passed multiple times in case there
		are multiple files to inject. See connection profile examples and reference at:
		https://www.balena.io/docs/reference/OS/network/2.x/
		https://developer.gnome.org/NetworkManager/stable/nm-settings.html
```

Reply:
@pdcastro nice wording, I'm happy with that :+1: